src/cli/existing_mnemonic.rs

```diff
@@ -79,6 +81,16 @@ pub fn subcommand<'a, 'b>() -> App<'a, 'b> {
                 and ^(0x00[a-fA-F0-9]{62})$ for BLS withdrawal credentials.",
                 ),
         )
+        .arg(
+            Arg::with_name("kdf")
+                .long("kdf")
+                .required(false)
+                .takes_value(true)
+                .help(
```

Comment:
I believe it's also useful to specify possible values here via `.possible_values()` as described in https://github.com/clap-rs/clap/blob/v2.34.0/examples/11_only_specific_values.rs#L21

Reply:
added in [9ea991f](https://github.com/ChorusOne/eth-staking-smith/pull/18/commits/9ea991ffa7318b5027c07b6d17421f0e83be3f23)